extensions/collection-assets/README.md

```diff
@@ -5,6 +5,7 @@
 - **Field Name Prefix: -**
 - **Scope: Collection**
 - **Extension [Maturity Classification](../README.md#extension-maturity): Proposal**
+- **Owner**: @m-mohr
```

Comment:
Should we add links? The @... don't link automatically in markdown, it seems.

Reply:
How do you think people will use the owner info here? I thought it would be most helpful for figuring out who to `@` in gitter / in issues -- I don't think a GitHub profile link would help much there

Reply:
Yeah, indeed. As long as you can't dm people here, it won't help too much, indeed.